agreement_rebate/models/agreement.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Tecnativa - Carlos Dauden
+# Copyright 2020 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models, fields
+
+
+class Agreement(models.Model):
+    _inherit = 'agreement'
+
+    def _selection_rebate_type(self):
+        return [
+            ('global', 'Global (A discount global for all lines)'),
+            ('line', 'Line (A discount for every line)'),
+            ('section_total', 'Compute total and apply discount rule match'),
+            ('section_prorated', 'Compute multi-dicounts by sections amount'),
+        ]
+
+    rebate_type = fields.Selection(
+        selection=lambda self: self._selection_rebate_type(),
+        string='rebate type',
```

Comment:
This is not needed as it's auto-built, but if explicit, put first in upper:
```suggestion
        string='Rebate type',
```